pallets/did/src/did_details.rs

```diff
@@ -612,28 +610,37 @@ impl<T: Config> DidDetails<T> {
 	}
 }
 
-pub(crate) type DidNewKeyAgreementKeySet<T> = BoundedBTreeSet<DidEncryptionKey, <T as Config>::MaxNewKeyAgreementKeys>;
-pub(crate) type DidKeyAgreementKeySet<T> = BoundedBTreeSet<KeyIdOf<T>, <T as Config>::MaxTotalKeyAgreementKeys>;
-pub(crate) type DidPublicKeyMap<T> =
-	BoundedBTreeMap<KeyIdOf<T>, DidPublicKeyDetails<BlockNumberOf<T>>, <T as Config>::MaxPublicKeysPerDid>;
+type DidNewKeyAgreementKeySet<MaxNewKeyAgreementKeys> = BoundedBTreeSet<DidEncryptionKey, MaxNewKeyAgreementKeys>;
+#[cfg(any(feature = "runtime-benchmarks", test))]
+pub(crate) type DidNewKeyAgreementKeySetOf<T> = DidNewKeyAgreementKeySet<<T as Config>::MaxNewKeyAgreementKeys>;
```

Comment:
The type is used in two files (mock_utils.rs and tests.rs). So not to implement it in the two different files I think it is better to let it there.

Reply:
You can still declare it once, and import it in the other file. But it does not belong to the `did_details.rs` file, since that is not feature-gated. The `mock_utils` module is already feature-gated with `#[cfg(any(feature = "runtime-benchmarks", test))]`, so that's ok. Declare it there, and import it in the tests. That should work 😊

Reply:
Resolved in: [1562f32](https://github.com/KILTprotocol/kilt-node/pull/530/commits/1562f325a85e915e846d0886010d27c3787bb064)